MdePkg/Library/StackCheckLib/Readme.md

```diff
@@ -195,20 +195,9 @@ This will cause `MdeLibs.dsc.inc` to not link `StackCheckLibNull` and instead li
 stack cookie updating versions of `DxeCoreEntryPoint`, `StandaloneMmDriverEntryPoint`,
 `UefiApplicationEntryPoint`, and `UefiDriverEntryPoint`.
 
-Because edk2 does not implement exception handling for `SEC` and `PEI_CORE`, `MdeLibs.dsc.inc`
-uses `StackCheckLibNull` for these phases always. As a result, dynamic stack cookies are also

```

Comment:
I think we should keep the section on StackCheckLib vs StackCheckLibNull, that still applies to these phases, even without dynamic stack cookies, because they will still have static stack cookies and stack cookie checking is orthogonal to how the cookie value was chosen. The part about exception handling for SEC and PEI_CORE was also a requested addition from Mike Kinney who pointed out platforms could implement their own exception handling there.

Reply:
I'll tweak this, but I'll note that the prose about exception handling is duplicated here so I'm not sure it needs to be repeated.